openupgrade_scripts/scripts/stock/16.0.1.1/post-migration.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2023-Today GRAP (http://www.grap.coop)
+# @author Sylvain LE GAL (https://twitter.com/legalsylvain)
```

Comment:
Please don't create 2 lines for the same autorship. Only one with both credits.

Reply:
No Sorry.
1) that's matter of taste, not on OCA rule.
2) that is the pattern used in the 170 GRAP modules under OCA umbrella, so changing all the headers requires time and doesn't provide any value.

regards.

Reply:
I'm not caring right now about the other 170 modules. I'm caring about this file in OpenUpgrade, and nobody uses a copyright double line, so please change it at least here.

Reply:
> I'm not caring right now about the other 170 modules. 

If I do this for this PR, you will feel legitimate to ask me on my other PRs, about the other 170 contributions, and it will start to represent a significant amount of work time and noise for me, and unnecessary time (for me & the OCA).

> and nobody uses a copyright double line, so please change it at least here.

Except :
Acsone : https://github.com/OCA/mis-builder/blob/16.0/mis_builder/models/accounting_none.py#L1
Akretion : https://github.com/OCA/l10n-france/blob/16.0/l10n_fr_account_tax_unece/__manifest__.py#L1
coopiteasy : https://github.com/OCA/cooperative/blob/14.0/cooperator/models/account_journal.py#L1
etc.

Pedro, I greatly appreciate the quality of your remarks in your many reviews and I learn a lot from reading you. But from time to time I think you confuse personal preferences with OCA rules.
This is also the case when you ask contributors to squash their commits (which are not fixups), or to remove empty lines in a function. These are developers' preferences. Everyone has their own, and it doesn't matter if several ways are present in the OCA.
I sincerely think that the level for contributing to the OCA is already very high (pre-commit, CI, keep history when migrating, don't push translation through github but through weblate, etc.)
I don't think that discussing this kind of things brings anything positive. it's a bit of a nitpick. And at some level, this may cause some contributors to become discouraged.

Reply:
I'm saying here in OpenUpgrade. Where do you find double line here in OU? 

And it's not nitpicking. It's for not hightlighting your contribution over others. Why do you deserve 2 lines meanwhile other contributors only put one?

And yes, it's already stated in OCA template: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L1

Reply:
> I'm saying here in OpenUpgrade. Where do you find double line here in OU?

Openupgrade is an OCA repo, as others. What you ask is not an OCA rule.

> It's for not hightlighting your contribution over others. Why do you deserve 2 lines meanwhile other contributors only put one?

I don't want to highlight anything. A line is for the company (copyright), another one is for the contributor (author). That's 2 differents things. [See my previous comment](https://github.com/OCA/OpenUpgrade/pull/3818#discussion_r1162011842). Other OCA contributors does the same, and there is no rules for that topics.

In fact, in contributors file, Tecnativa contributors "consumes" [two lines](https://github.com/OCA/reporting-engine/blob/16.0/base_comment_template/readme/CONTRIBUTORS.rst) and GRAP (and other OCA) contributors [one line](https://github.com/OCA/reporting-engine/blob/16.0/sql_request_abstract/readme/CONTRIBUTORS.rst).
 
It's a matter of taste. I don't think Tecnativa set two lines in the ``CONTRIBUTORS``  to be "highlighted". Do you ? 

> And yes, it's already stated in OCA template: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L1

A template is a proposal. not a guideline.

> And it's not nitpicking

Yes, definitely. And this subject is quite boring. let's move on please.

Reply:
Sorry, the same as you have set other red lines, for me this is one. It's not a matter of taste. It's a sign of respect about dedicating 2 lines for your copyright instead of 1.